### ISSUE TITLE
Issue: Remote Scanning Function Needs Updating

### ISSUE THREAD

**Issue by KitenaLittlepaws** [OpenCollarTeam/OpenCollar#857]
What version of OpenCollar are you using? 8.2.1

What behavior did you expect? Remote scanning to pick up only those that are available to connect with.

What behavior did you see instead? Remote scanning picked up everyone in proximity instead.

What steps does someone need to take to reproduce the problem? Scan a room full of people with one or more people not even wearing a collar.  It makes it look as though the person not wearing a collar is wearing one.

**Comment by Medea-Destiny** [comment]
Are you referring to connecting to a collar via the owner hud?

**Comment by KitenaLittlepaws** [comment]
I am referring to using the scanning function on the remote HUD.  It shows
everyone in proximity rather than just those One can connect to.

**Comment by KitenaLittlepaws** [comment]
At the very least the scanning function should only show those wearing a
collar instead of everyone in proximity.  I can look in the area and see
who is in the area if I want to know that.

**Comment by SilkieSabra** [comment]
Yes the owner hud now called the Remote, when you click the People icon at the bottom, shows the entire list of nearby avatars.  You have to know which collars in the room you have access to, or else if you are fishing for a collar on public access, try each in turn.  Once you have connected, you can add them to Favorites (star icon).  You can then use that list instead of Nearby to connect.

**Comment by KitenaLittlepaws** [comment]
Ok.  Here is the issue.  Someone that does not wear a collar at all shows
up on said scan.  This is a problem.  It makes it look like everyone around
is wearing one.  Can the remote not detect a collar?  If not then it needs
improvement to be able to do so.

**Comment by Medea-Destiny** [comment]
The way to detect collars would be to send a message to the interface channel for each user found and await a response, implementing a delay from pressing the button to the menu appearing of perhaps 10-20 seconds to ensure that all available collars have had a chance to reply. I don't think that this is the preferable option.

**Comment by KitenaLittlepaws** [comment]
That's interesting because there are remotes that don't work with the
version 8.2.1 collar that can at least detect a if someone has access to a
collar or if they have capture/kidnap mode turned on only.
